test/core/componentGroupTests.ts

```diff
@@ -164,32 +126,54 @@ describe("ComponentGroups", () => {
     assert.lengthOf(cg.components(), 0, "cg has no components");
   });
 
-  describe("ComponentGroup._requestedSpace works as expected", () => {
-    it("_works for an empty ComponentGroup", () => {
-        var cg = new Plottable.Component.Group();
-        var request = cg._requestedSpace(10, 10);
-        verifySpaceRequest(request, 0, 0, false, false, "");
+  describe("requests space based on contents, but occupies total offered space", () => {
+    var SVG_WIDTH = 400;
+    var SVG_HEIGHT = 400;
+
+    it("with no Components", () => {
+      var svg = generateSVG();
+      var cg = new Plottable.Component.Group([]);
+
+      var request = cg._requestedSpace(SVG_WIDTH, SVG_HEIGHT);
+      verifySpaceRequest(request, 0, 0, false, false, "empty Group doesn't request any space");
+
+      cg.renderTo(svg);
+      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      svg.remove();
     });
 
-    it("works for a ComponentGroup with only proportional-size components", () => {
-      var cg = new Plottable.Component.Group();
-      var c1 = new Plottable.Component.AbstractComponent();
-      var c2 = new Plottable.Component.AbstractComponent();
-      cg.below(c1).below(c2);
-      var request = cg._requestedSpace(10, 10);
-      verifySpaceRequest(request, 0, 0, false, false, "");
+    it("with proportional Components", () => {
+      var svg = generateSVG();
+      var cg = new Plottable.Component.Group([
+        new Plottable.Component.AbstractComponent(),
+        new Plottable.Component.AbstractComponent()
+      ]);
+
+      var request = cg._requestedSpace(SVG_WIDTH, SVG_HEIGHT);
+      verifySpaceRequest(request, 0, 0, false, false, "empty Group doesn't request any space");
+
+      cg.renderTo(svg);
+      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      svg.remove();
     });
```

Comment:
I should probably assert something about `_isFixedWidth()`. It doesn't request space, but it's non-fixed so it will accept more.

The problem is that this behavior really manifests when inside a `Table`, but that's more of an integration test.